discovery-infra/start_discovery.py

```diff
@@ -618,6 +647,11 @@ def main():
     parser.add_argument(
         "-ps", "--pull-secret", help="Pull secret", type=str, default=""
     )
+    parser.add_argument(
```

Comment:
Probably the documentation is worth updating with this flag and an example of setting configuring static IPs.